### DISCUSSION TITLE
Grid view truncate numeric within 3 digits after the point

### DISCUSSION THREAD

**Discussion by KVentrue** [dbeaver/dbeaver#24] [Q&A] [answered]
e.g. Grid view just shows 11.234 for 11.23456789 in result set. Text mode works fine. Pls help locate the problem and maybe I can fix it.:)

**Comment by serge-rider** [comment, marked as answer]
It is not a bug. Go to Preferences -> Common -> Result Sets -> Data Formats. Select "Numbers" and set "Maximum fraction digits" to some appropriate value.

**Comment by KVentrue** [comment]
Got it. Many thanks!
